data/json/items/obsolete.json

```diff
@@ -481,6 +481,7 @@
     "durability": 8,
     "min_cycle_recoil": 1350,
     "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "delete": { "weapon_category": [ "AUTOMATIC_RIFLES" ] },
```

Comment:
You probably don't need this line since the items are obsolete.

Reply:
Or are you saying it's necessary to prevent it from showing up as a valid weapon in the martial art?

Reply:
I added that line because the obsoleted gun was showing up in the Krav Maga list as IF YOU SEE THIS IT IS A BUG